charts/k8s-watcher/values.yaml

```diff
@@ -71,6 +71,7 @@ watcher:
   allowReadingPodLogs: true
   enableHelm: true
   daemon:
```

Comment:
instead of duplicating the boolean value in two separate places in the config, can't we just refer to the same YAML path when we read the configuration?

Reply:
I see thanks for explaining